e2e/index.js

```diff
@@ -0,0 +1,20 @@
+require('dotenv').config({ path: process.env.DOTENV_CONFIG_PATH });
+
+console.log('');
+console.log('Console.log in index.js');
+
+const u = process.env.USER_NAME.split('@');
+console.log(`USER_NAME: ${u}`);
+
+const r = process.env.READER_USER.split('@');
+console.log(`READER_USER: ${r}`);
+
+const w = process.env.WRITER_USER.split('@');
+console.log(`WRITER_USER: ${w}`);
+
+const c = process.env.COLLABORATOR_USER.split('@');
+console.log(`COLLABORATOR_USER: ${c}`);
+
+const a = process.env.ACCESS_TEST_USER.split('@');
+console.log(`ACCESS_TEST_USER: ${a}`);
+console.log('');
```

Comment:
Console log users in case someone needs to know about them. I can remove log if we don't need to do that.

<img width="1497" alt="Screen Shot 2021-08-19 at 4 21 57 PM" src="https://user-images.githubusercontent.com/35533885/130138652-17217a12-afeb-4310-a105-a1e120a81f3f.png">

Reply:
It's nice having this in the logs. Alternatively, should we just log the user when we're signing in (if we don't already?) - that would probably be sufficient

Reply:
Cool, so I'll keep the log of user emails. CircleCI will also mask emails with `******` if log is at the sign-out.